rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -924,6 +930,96 @@ void saveFederationElection_shouldSaveInStorage() {
         assertArrayEquals(federationElectionSerializedWithVote, actualAbiCallElectionSerialized);
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("save fund transaction unsigned hash tests")
+    class SaveFundTransactionUnsignedHashTests {
+
+        StorageAccessor storageAccessor;
+        FederationStorageProvider federationStorageProvider;
+        ActivationConfig.ForBlock activations;
+
+        @BeforeEach
+        void setup() {
+            storageAccessor = new InMemoryStorage();
+            federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+            activations = ActivationConfigsForTest.all().forBlock(0L);
+        }
+
+        @Test
+        void saveFundTransactionUnsignedHash_preLovell700_shouldNotSaveInStorage() {
+
+            // Arrange
+
+            ActivationConfig.ForBlock arrowheadActivations = ActivationConfigsForTest.arrowhead600().forBlock(0L);
+
+            Sha256Hash expectedHash = BitcoinTestUtils.createHash(123_456_789);
+
+            federationStorageProvider.setFundTransactionUnsignedHash(expectedHash);
+
+            // Act
+
+            federationStorageProvider.save(networkParameters, arrowheadActivations);
+
+            // Assert
+
+            Sha256Hash actualHash = storageAccessor.getFromRepository(SVP_FUND_TX_HASH_UNSIGNED.getKey(), BridgeSerializationUtils::deserializeSha256Hash);
+
+            assertNull(actualHash);
+
+        }
+
+        @Test
+        void saveFundTransactionUnsignedHash_postLovell700_shouldSaveInStorage() {
```

Comment:
no big deal, but im not sure if the _postLovell_ word adds value. I find it a bit redundant

Reply:
Why? When other forks are added, we won't have to update this.
If we don't use `postLovell700` because this is the latest fork, then, when more forks are added in the future, the name of this test would't express what fork it is testing.
I think it's a good idea to keep it, and also, this way we keep it consistent with other tests.